packages/artifact/__tests__/retry.test.ts

```diff
@@ -3,7 +3,7 @@ import * as net from 'net'
 import * as core from '@actions/core'
 import * as configVariables from '../src/internal/config-variables'
 import {retry} from '../src/internal/requestUtils'
-import {IHttpClientResponse} from '@actions/http-client/interfaces'
+import {IHttpClientResponse} from '@actions/http-client/lib/interfaces'
```

Comment:
Having to put `lib` in the module reference seems to be a `lerna` thing.  I tested what happens if you install the actual package through `npm` and it doesn't change.

Other packages in this repo reference each other's submodules through `lib/`.